src/api/cron/index.ts

```diff
@@ -0,0 +1,26 @@
+import * as cron from 'node-cron';
+import { CronSchedulerJob } from '../types';
+import { asyncTaskToSyncTask, taskWithErrorLogging } from '../helpers/cron/scheduler';
+import { Context } from '.keystone/types'; // eslint-disable-line
+
+/**
+ * cronJobScheduler
+ * Validates and sets up cron jobs to run
+ * @param {CronSchedulerJob[]} jobs: cron jobs to schedule to run
+ * @param {Context} KeystoneJS context API
+ */
+export const cronJobScheduler = (jobs: CronSchedulerJob[], context: Context) => {
+  jobs.forEach((job) => {
+    const { cronExpression, description, task } = job;
```

Comment:
Could we have a `Job` type? Instead of `CronSchedulerJob[]`, the `job` in the `forEach` could also consume this

Reply:
`export default`?